ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -1161,6 +1164,33 @@ ORK_CLASS_AVAILABLE
 @interface ORKTextAnswerFormat : ORKAnswerFormat
 
 /**
+ Returns an initialized text answer format using the regular expression.
+ 
+ This method is one of the designated initializers.
+ 
+ @param expression                The regular expression used to valide the text.
+ @param description               The description of the valid input presented to the user upon invalid input.
+ 
+ @return An initialized validated text answer format.
+ */
+- (instancetype)initWithValidationExpression:(nullable NSString *)expression
+                       validInputDescription:(nullable NSString *)description NS_DESIGNATED_INITIALIZER;
+
```

Comment:
It is nice to have all the initializers together just like other answer formats.